bqplot/pyplot.py

```diff
@@ -816,6 +816,30 @@ def bar(x, y, **kwargs):
     kwargs['y'] = y
     return _draw_mark(Bars, **kwargs)
 
+def boxplot(x, y, **kwargs):
+    """Draws a boxplot in the current context figure.
+
+    Parameters
+    ----------
+
+    x: numpy.ndarray, 1d
+        The x-coordinates of the data points.
+    y: numpy.ndarray, 2d
+        The data from which the boxes are to be created. Each row of the data
+        corresponds to one box drawn in the plot.
+    options: dict (default: {})
+        Options for the scales to be created. If a scale labeled 'x' is
```

Comment:
Shouldn't we customize this doc to the mark itself?

Reply:
you mean move to this to the base class itself? didn't completely follow.